ajax/libs/jquery.x18n/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "jquery.x18n",
+  "filename": "jquery.x18n.min.js",
+  "version": "1.1.3",
+  "description": "The jQuery adapter for the x18n internationalisiation library.",
+  "author": {
+    "name": "Florian H.",
+    "url": "https://github.com/js-coder"
+  },
+  "keywords": [
+    "i18n",
+    "jquery",
+    "x18",
+    "javascript",
+    "internationalization"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/florian/jQuery.x18n.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/florian/jQuery.x18n.git",
+    "fileMap": [
+      {
+        "basePath": "lib",
+        "files": [
+          "jquery.x18n.js"
```

Comment:
Let's use more flexible one? Such as `jquery.x18n!(*.coffee)`?

Reply:
I thought we use included mode when `lib` as the basePath, `!` is excluded mode.
I am a little confuse in this pattern.
Using `!` as excluded mode is good when `lib` as the basePath?
Thanks

Reply:
Sorry for the mistake. @sashberd Please use `jquery.x18n*.+(js|css|map)`. Thanks.